OxfordBodleian/Juel-Jensen/BDLaethb6.xml

```diff
@@ -40,9 +43,8619 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     <msContents>
                         <summary/>
                         
-                        <!--<msItem xml:id="ms_i1">
-                      
-                        </msItem>-->
+                        <msItem xml:id="ms_i1.1">
+                            <locus from="1ra" to="1vb"/>
+                            <title type="incomplete" ref="LIT4035SenkessarDL#Terr"/>
+                            <note><locus target="#1"/> contains the beginning of 1st Ṭǝrr, written by the same hand as the rest of the manuscript,
+                                which is also contained in this manuscript on <locus target="#126"/>. It is unclear
+                            whether the leaf has been displaced or belonged originally to another manuscript.</note>
```

Comment:
It is a rather puzzling question!
In my opinion, f.1 belonged to another manuscript since along the inner margin are sets of holes that do not correspond to the present sewing pattern. However, looking closely, it is possible to note further holes (partially covered by the parchment guard) that do match the present pattern.
I think that the f. 1 belonged to another manuscript (possibly slightly smaller) and for some reason it was selected to become part of the present one. The lower portion of blank parchment was added to repair the leaf and adjust the format, then the leaf was sewn to this manuscript. Later it must have become detached, the parchment guard has been added and the leaf has been linked to the sewing, in the present position.
It is just an hypothesis but it can work. However, the mending and the "links" around the sewing stations look modern to me.